spec/unit/virtus/attribute/set_default_value_spec.rb

```diff
@@ -3,7 +3,16 @@
 describe Virtus::Attribute, '#set_default_value' do
   let(:object) { described_class.build(String, options.merge(:name => name, :default => default)) }
 
-  let(:model)    { Class.new { def name; 'model'; end; attr_reader :test } }
```

Comment:
Personally I find it harder. Also very difficult to see diffs in 1-liners as code changes. Accent on "personally".